lib/wit/gitrepo.py

```diff
@@ -89,6 +95,17 @@ def clone(self, source, name):
             else:
                 raise
 
+    # Only newer git versions can use '--reference-if-able', so we emulate the 'if-able'.
+    def git_reference_capable(self):
+        env_ref_path = os.getenv("WIT_WORKSPACE_REFERENCE")
+        paths = [Path(env_ref_path) / self.name,
+                 Path(env_ref_path) / (self.name+'.git')]
+        for path in paths:
+            log.warn("{} {}".format(path, os.path.isdir(path)))
+            if os.path.isdir(path):
+                return True, ["--reference", str(path), "--dissociate"]
+        return False, []
```

Comment:
This looks like a Go-ism. In Python, you would just return either an empty or non-empty list, and you can just put the returned list directly in the `if` statement, since empty lists are falsey and non-empty lists are truthy.

In addition, in this specific case you could actually just `.extend()` the return values whether or not it's empty since the `.extend()` method is a no-op if you're extending with an empty list.